cculkin8/cc-capstone
# Cc holding

# Description
There have been several changes to this repository, user's own profiles are now only visible to them. Switched from tinylink to microlink to show the actual map. Login field has been slightly changed, users can like posts. The homepage has the top 3 posts showing and has a "feeling lucky" button that pulls a completely random post from the database for them to go to.
Fixes # (issue)
Link preview is now working, which is good. Users also don't see other people's profiles when they click "profile"

# Testing Instructions
Please create a profile, login, create a post. Like it, then check the homepage. It should appear under the "top posts" section. Also check the random button. 
# Checklist:
